compiler/test/formatter_outputs/comments.gr

```diff
@@ -77,7 +77,7 @@ if (a /*why*/) {
   // end block
 }
 
-let myfun /*special*/ = (/*lead*/ x, /*follow*/ y) => x + 5
+let myfun /*special*/ = (/*lead*/ x /*follow*/, y) => x + 5
```

Comment:
Why did you change the output locations of this?

Reply:
Because the comma isn't in the AST, the choice of where to place it in relation to the comments is arbitrary, and the new way of processing chooses the other.

Reply:
Why does it choose that? I prefer the old way.

Reply:
Before I was looking for leading comments, but that broke how mixed comments and separators where we need to consider trailing comments.

This way, this formats better:  50% of one, half the other IMO

`let myfun /*special*/ = (x /*lead*/, y /*follow*/) => x + 5`

Reply:
Ah, so it's actually *better* at keeping the input location now. Nice!